exonum-java-binding/packaging/pom.xml

```diff
@@ -0,0 +1,133 @@
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>exonum-java-binding-packaging</artifactId>
+  <version>0.5-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <properties>
+    <packaging.basedir>${project.parent.basedir}/core/rust/target/prepackage</packaging.basedir>
+    <packaging.rust>${project.parent.basedir}/core/rust</packaging.rust>
+    <packaging.java>${project.parent.basedir}/core/rust/target/debug/lib/java</packaging.java>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-java-binding-core</artifactId>
+      <version>0.5-SNAPSHOT</version>
+      <type>pom</type>
+    </dependency>
+
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-time-oracle</artifactId>
+      <version>0.5-SNAPSHOT</version>
+      <type>pom</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin><!--Do not create Jar-->
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-jar</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin><!--Do not install Jar-->
+        <artifactId>maven-install-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-install</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>app-packaging</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <configuration>
+              <includeScope>runtime</includeScope>
+            </configuration>
+            <executions>
+              <execution>
+                <id>copy-core-and-time-oracle-jars</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>copy</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>${packaging.java}</outputDirectory>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>com.exonum.binding</groupId>
+                      <artifactId>exonum-java-binding-core</artifactId>
+                      <version>0.5-SNAPSHOT</version>
+                    </artifactItem>
+                    <artifactItem>
+                      <groupId>com.exonum.binding</groupId>
+                      <artifactId>exonum-time-oracle</artifactId>
+                      <version>0.5-SNAPSHOT</version>
+                    </artifactItem>
+                  </artifactItems>
+                </configuration>
+              </execution>
+              <execution>
+                <id>copy-java-dependencies</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>copy-dependencies</goal>
```

Comment:
> Goal that copies the project dependencies from the repository to a defined location.

Why then we copy core and time-oracle explicitly above?

Reply:
Because we use `type=pom` dependencies, and it seems like in this case `copy-dependencies` will not copy jars. Will be fixed later, with ECR-3039